test/functional/geckodriver-e2e-specs.js

```diff
@@ -0,0 +1,48 @@
+// transpile:mocha
```

Comment:
do these test work on CI?

Reply:
nope, emulators won't have firefox, we'll need to skip them for now.

Reply:
I mean we could preinstall Firefox from apk downloaded from the internet in the suite setup section

Reply:
Gonna check if it's easy enough to build firefox for x86 emulators or find an apk online

Reply:
Firefox devs shared the official archives to where we can download the apks. https://archive.mozilla.org/pub/mobile/releases/68.5b3. Gonna try to enable CI on a next iteration